### DISCUSSION TITLE
已安装新版1.6.5 (6287) ，仍提示需要更新？

### DISCUSSION THREAD

**Discussion by Leojc** [LSPosed/LSPosed#1500] [Q&A] [answered]
提示为:
1.6.5 (6287) - Zygisk
请安装新版 LSPosed

当前最新就是6287

**Comment by yujincheng08** [comment, marked as answer]
zygisk 都没发布，何来最新版。

**Reply by Leojc** [reply]
哦哦，我用的actions里的测试版。那就是说测试版的更新提示是不准的，不用管对吗？我以为是出bug了
